app/models/term.rb

```diff
@@ -2,4 +2,8 @@
 
 class Term < ApplicationRecord
   has_and_belongs_to_many :phrases
+  validates :word, :meaning, :reading, presence: true
```

Comment:
Write test for this please
```ruby
# spec/models/term_spec.rb
it { should validate_presence_of :word }
# ...
```

See more at http://matchers.shoulda.io/docs/v4.1.1/